HACKING.md

```diff
@@ -0,0 +1,32 @@
+# Hacking on `rss2email`
+
+This document is still TODO. Please contribute to it and file issues
+if you have a question (or something is not described sufficiently).
+
+## Using nix support
+
+`rss2email` has a few nix definitions in order to simplify development.
+In order to use them you need to install the [nix package
+manager][https://nixos.org/nix] version 2 or later on your system.
+
+### Open a shell with all dependencies
+
+Run `nix-shell` in the top directory. I will open a bash with all
+dependencies (python and system) you need for developing `rss2email`.
+This uses the `shell.nix` file.
+
+### Test `rss2email` against multiple python versions
+
+`nix/release.nix` contains an expression to build rss2email with
+different versions of python and run its tests suite with that
+version.
+
+So far, Python 3.5, 3.6 and 3.7 are supported.
+
+You can build them like this:
+
+```
+nix-build -A rss2email-3-5 nix/release.nix
+nix-build -A rss2email-3-6 nix/release.nix
+nix-build -A rss2email-3-7 nix/release.nix
+```
```

Comment:
Nit: add something like “You can build them all in one command with `nix-build nix/release.nix`”.

Also, the attributes would maybe be better named `rss2email-python3-5` etc.? Right now it reads as though building `rss2email`'s version `3.5` etc.